lib/Build.js

```diff
@@ -44,8 +44,9 @@ const Account = require('./Account');
 const BuilderConfig = require('./util/BuilderConfig');
 const ProjectConfig = require('./util/ProjectConfig');
 const Builder = require('Builder');
+const BuilderLifecycle = require('./util/BuilderLifecycle');
```

Comment:
I'm not really familiar with the Builder internals and why this wasn't included/used already, but I'm going to consider this okay as you've been hand testing :)

Reply:
It looks like some of the logic was pulled into this new util file that was added, it's a lot neater!

Reply:
Yes, Nasib is right. This is a new file I introduced to pull out some common code now that the post build is happening in both tests and regular deploy/run code.